spec/integrations/rebound_bracket_spec.rb

```diff
@@ -0,0 +1,14 @@
+describe 'rebinding [' do
+  context 'initialized before sourcing the plugin' do
+    before do
+      session.send_string("function [ { test \"$@\" }")
```

Comment:
I think I just realized that this is probably the problem - there will be an additional argument (`]`) in most of these cases. I'll get this fixed.